Ports/isl/package.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=isl
+version=0.24
+useconfigure="true"
+use_fresh_config_sub=true
```

Comment:
Maybe we should `shellcheck` the `package.sh` files to force more consistent use of '"'.